Primo.js

```diff
@@ -251,48 +253,36 @@ function importPNX(text) {
 	item.title = ZU.xpathText(doc, '//display/title');
 	if(item.title) item.title = ZU.unescapeHTML(item.title);
 	
-	var creators;
-	var contributors;
-	if(ZU.xpathText(doc, '//display/creator')) {
-		creators = ZU.xpath(doc, '//display/creator'); 
-	}
-	
-	if(ZU.xpathText(doc, '//display/contributor')) {
-		contributors = ZU.xpath(doc, '//display/contributor'); 
-	}
-	
-	if(!creators && contributors) { // <creator> not available using <contributor> as author instead
+	var creators = ZU.xpath(doc, '//display/creator');
+	var contributors = ZU.xpath(doc, '//display/contributor');
+	if(!creators.length && contributors.length) {
+		// <creator> not available using <contributor> as author instead
 		creators = contributors;
-		contributors = null;
-	}
-	
-	if(!creators && ! contributors){
-		creators = ZU.xpath(doc, '//addata/addau')
+		contributors = [];
 	}
 	
-	for(i in creators) {
-		if(creators[i]) {
-			var creator  = ZU.unescapeHTML(creators[i].textContent).split(/\s*;\s*/);
-			for(j in creator){
-				creator[j] = creator[j].replace(/\d{4}-(\d{4})?/g, '');
-				item.creators.push(Zotero.Utilities.cleanAuthor(creator[j], "author", true));
-			}			
+	// //addata/addau is great because it lists authors in last, first format,
+	// but it can also have a bunch of junk. We'll use it to help split authors
+	var splitGuidance = {};
+	var addau = ZU.xpath(doc, '//addata/addau');
```

Comment:
We need both here:
```
ZU.xpath(doc, '//addata/addau|//addata/au')
```
AFAIK: The `au` is refering to the creators (authors) and the `addau` is refering to the contributors (additional authors). However, we shouldn't count on that and it should be IMO safe to store both kind of creators/contributors in the same data structure, because we will use it only as a split guidance (lookup table).